src/sparse_dd.ml

```diff
@@ -65,6 +69,9 @@ let options =
     "size", Arg.String (fun x -> size := Int64.of_string x), (fun () -> Int64.to_string !size), "number of bytes to copy";
     "prezeroed", Arg.Set prezeroed, (fun () -> string_of_bool !prezeroed), "assume the destination disk has been prezeroed";
     "machine", Arg.Set machine_readable_progress, (fun () -> string_of_bool !machine_readable_progress), "emit machine-readable output";
+    "ssl-legacy", Arg.Set ssl_legacy, (fun () -> string_of_bool !ssl_legacy), " ssl legacy flag";
```

Comment:
Descriptions:
ssl-legacy: "for TLS, allow all protocol versions instead of just TLSv1.2"
good-ciphersuites: "the list of ciphersuites to allow for TLS"
legacy-ciphersuites: "additional TLS ciphersuites allowed only if ssl-legacy is set"